http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -53,14 +78,81 @@ class GKEInterpreter[F[_]: Parallel: ContextShift](
       getApp <- F.fromOption(getAppOpt, AppNotFoundException(msg.project, msg.appName, ctx.traceId))
       gkeClusterId = getApp.cluster.getGkeClusterId
       namespaceName = getApp.app.appResources.namespace.name
+      googleProject = getApp.cluster.googleProject
+      gsa = getApp.app.googleServiceAccount
+      ksaName = KubernetesServiceAccountName(s"${namespaceName.value}-ksa")
```

Comment:
I think the namespace name is `${appName}-namespace` so this would end up being ${appName}-namespace-ksa`. Maybe just `${appName}-ksa` would be cleaner?

Reply:
hmmm yeah I can update. Is app name unique in a cluster?